briefcase-api.rst

```diff
@@ -21,7 +21,7 @@ The APIs used:
 
 1. OpenRosa Form Discovery API
 2. download forms and media files using URLs form discovery.
-3. ``view/submissionList`` to obtain a chunk of submission keys.
+3. ``view/submissionlist`` to obtain a chunk of submission keys.
```

Comment:
I'm not 100% sure that the API requires it, but to be safe, it should be case-sensitive. This change should be reverted.